app/assets/stylesheets/general.scss

```diff
@@ -8,3 +8,18 @@ html {
 h2 {
   display: inline;
 }
+
+body {
+  background: image-url('bg_photo.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size